packages/design-tokens/src/css/dark-theme-colors.css

```diff
@@ -41,8 +41,8 @@
   --color-border-default: var(--brand-colors-grey-grey400);
   /* Muted color for borders (#858b9a33) */
   --color-border-muted: #858b9a33;
-  /* Default color for overlays (scrim) (black-40%) */
-  --color-overlay-default: #00000066;
+  /* Default color for overlays (scrim) (#3f434a99) */
+  --color-overlay-default: #3f434a99;
```

Comment:
<img width="651" alt="Screenshot 2025-05-20 at 1 26 47 PM" src="https://github.com/user-attachments/assets/6e14e567-8eae-446c-b701-e35d2714e08d" />

Reply:
@Akatori-Design can you check this is right please?